DOMjudge_tag_release.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+. release.env
+
+for i in `git tag | grep 7`; do git tag -d $i; done
```

Comment:
What does this do?

Reply:
make that locally I can test changes with the tags, should not be in the final script.